app/templates/main/browse-consignment.html

```diff
@@ -60,14 +60,6 @@
                                     {% endif %}
                                 </td>
                             </tr>
-                            <tr class="govuk-table__row govuk-table__row__browse--mobile">
```

Comment:
was this dead code?

Reply:
yes, that code was not required for mobile view